src/App.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import Header from './components/Header/Header'
+import MainArea from './components/MainArea/MainArea';
+import Footer from './components/Footer/Footer';
 
 function App() {
   return (
-    <h1>
-      Welcome to React App thats build using Webpack and Babel separately
-    </h1>
+    <>
+      <Header></Header>
+      <MainArea></MainArea>
+      <Footer></Footer>
+    </>
```

Comment:
You don't have to have closing tag if you not passing children so it can be self-closing.
<Header/> etc

Reply:
fixed